src/main/java/org/isf/examination/manager/ExaminationBrowserManager.java

```diff
@@ -77,17 +78,37 @@ public PatientExamination getDefaultPatientExamination(Patient patient) {
 	 * Get from last PatientExamination (only height, weight & note)
 	 */
 	public PatientExamination getFromLastPatientExamination(PatientExamination lastPatientExamination) {
-		PatientExamination newPatientExamination = new PatientExamination(new GregorianCalendar(), lastPatientExamination.getPatient(),
+		PatientExamination newPatientExamination = new PatientExamination(new GregorianCalendar(), 
+				lastPatientExamination.getPatient(),
 				lastPatientExamination.getPex_height(),
-				lastPatientExamination.getPex_weight(), lastPatientExamination.getPex_ap_min(), lastPatientExamination.getPex_ap_max(),
+				lastPatientExamination.getPex_weight(), 
+				lastPatientExamination.getPex_ap_min(), 
+				lastPatientExamination.getPex_ap_max(),
 				lastPatientExamination.getPex_hr(),
-				lastPatientExamination.getPex_temp(), lastPatientExamination.getPex_sat(), lastPatientExamination.getPex_hgt(),
-				lastPatientExamination.getPex_diuresis(), lastPatientExamination.getPex_diuresis_desc(), lastPatientExamination.getPex_bowel_desc(),
+				lastPatientExamination.getPex_temp(), 
+				lastPatientExamination.getPex_sat(), 
+				lastPatientExamination.getPex_hgt(),
+				lastPatientExamination.getPex_diuresis(), 
+				lastPatientExamination.getPex_diuresis_desc(), 
+				lastPatientExamination.getPex_bowel_desc(),
 				lastPatientExamination.getPex_rr(),
-				lastPatientExamination.getPex_auscultation(), lastPatientExamination.getPex_note());
+				lastPatientExamination.getPex_auscultation(), 
+				formatLastNote(lastPatientExamination));
 		return newPatientExamination;
 	}
 
+	private String formatLastNote(PatientExamination lastPatientExamination) {
+		String note = lastPatientExamination.getPex_note();
+		if (!note.isEmpty()) {
+			StringBuilder sbNote = new StringBuilder();
```

Comment:
I see you are adding new keys to the resource bundle why not use something like

`
MessageBundle.formatMessage("angal.examination.lastnote.fmt.msg", 
           TimeTools.formatDateTime(lastPatientExamination.getPex_date(), null), lastPatientExamination.getPex_note());
`

where angal.examination.lastnote.fmt.msg = "Last Note {0}:\n{1}\n\n".

Then you don't have to worry about word order and concatenation in other languages.

Reply:
you're right, I was just doing in the old fashined way... thanks!